testsuite/tests/kuadrant/observability/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""Conftest for Kuadrant observability tests"""
+
+import pytest
+from openshift_client import OpenShiftPythonException
+from openshift_client import selector
+
+from testsuite.httpx.auth import HttpxOidcClientAuth
+from testsuite.openshift.authorino import TracingOptions
+from testsuite.openshift.limitador import LimitadorCR
+from testsuite.policy.rate_limit_policy import Limit
+
+
+@pytest.fixture(scope="module")
+def auth(oidc_provider):
+    """Returns RHSSO authentication object for HTTPX"""
+    return HttpxOidcClientAuth(oidc_provider.get_token, "authorization")
+
+
+@pytest.fixture(scope="module")
+def rate_limit(rate_limit):
+    """Add limit to the policy"""
+    rate_limit.add_limit("basic", [Limit(5, 60)])
+    return rate_limit
+
+
+@pytest.fixture(scope="session")
+def limitador(kuadrant_system_client) -> LimitadorCR:
+    """Returns Limitador CR"""
+    try:
+        with kuadrant_system_client.context:
+            limitador = selector("limitador").object(cls=LimitadorCR)
+            limitador.committed = True
+    except OpenShiftPythonException:
+        pytest.fail("Running Kuadrant tests, but Authorino resource was not found")
+
+    return limitador
+
+
+@pytest.fixture(scope="module", autouse=True)
+def enable_tracing(authorino, limitador, tracing):
+    """Enable tracing for Authorino and Limitador"""
+    authorino.tracing = TracingOptions(tracing.collector_url, insecure=True)
+    limitador.tracing = TracingOptions(tracing.collector_url)
+    limitador.verbosity = 3
+    limitador.deployment.wait_for_ready()
```

Comment:
We should wait for limitador CR readiness not the deployment

Reply:
This will not work: https://github.com/Kuadrant/limitador-operator/issues/140